package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "clipboard-copy",
   "description": "Lightweight copy to clipboard for the web",
-  "version": "1.0.1",
+  "version": "1.0.2",
```

Comment:
Please don't change the package version in a PR. That is the job of the maintainer to do when they publish a new version. If someone else sends a PR that I merge and release first, then the version you set here will be wrong.